### DISCUSSION TITLE
FreshRSS is hard to use on mobile. Is there a workaround? (screenshot attached)

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5547] [Q&A]
I've set up some feeds but I want to be able to visit my FreshRSS server and read the articles from my mobile browser rather than an app. The UI works well on a tablet but is unusable on a mobile phone. Is there a solution to this? 

Edit: This screenshot is taken on a 5.4‑inch (diagonal) phone, so to understand how difficult this is to use, you might need to adjust your view if you are reading this on a larger device.
Edit 2: Photo graph for reference / scale:
![IMG_7479 Large](https://github.com/FreshRSS/FreshRSS/assets/1757768/dc5fd976-6caa-42f5-b8eb-24a71812fa91)

![IMG_271860CA3A3A-1](https://github.com/FreshRSS/FreshRSS/assets/1757768/0b4af589-b479-4762-8702-ff1196e4ec5c)

**Comment by beerisgood** [comment]
You can optimize it a little bit like this:
![image](https://github.com/FreshRSS/FreshRSS/assets/9625159/4daa28b4-9d84-4b38-a2c9-983534ccbc9a)

**Reply by ghost** [reply]
Thanks. That’s a little easier to view, but my fingers can’t use a UI that small. There are online readers that I could share my FreshRSS feed with. But then I wonder what the point of using FreshRSS is.

**Reply by Frenzie** [reply]
Thanks to @math-GH converting some values to CSS variables it should be fairly simple to increase the value of some paddings as a personal tweak. Maybe that could make sense as a kind of mini-theme?

Doing it indiscriminately would make me less happy. :-)

**Reply by ghost** [reply]
Could you say which file would I make those changes in? Also, do I need to restart the server or just refresh the page afterwards ? Thanks.

**Reply by Frenzie** [reply]
Just to see what it looks like, something like this in CustomCSS:

```css
@media (max-width: 840px) {
  :root {
    --frss-padding-flux-items: 1.5rem;
  }
}
```

Edit: from here: https://github.com/FreshRSS/Extensions

**Reply by ghost** [reply]
The result:
![Untitled](https://github.com/FreshRSS/FreshRSS/assets/1757768/5f3efb08-5bd7-4cf4-a204-c27668390dc5)

I think that a more extensive change would help. For example:
1. I think that the entire headline should be visible, facilitated by line wrapping. So not just the first few words. This could also be used to make each item two lines long instead of one, making them easier to tap on and easier to see.
2. The controls to star or mark as read should probably be hidden on small displays, or given a horizontal line of their own, separate from the title. The icon for the feed could be placed here too as well as the icon for the link to the source.

So I suppose what I'm looking for is an additional view mode to the existing ones.

**Reply by Frenzie** [reply]
1. has been my position for years but it's still slightly stuck on some minor development details. But in the meantime there's https://github.com/FreshRSS/Extensions/tree/master/xExtension-TitleWrap
2. I mostly disagree with since I find it convenient to have there. But it would make sense to extend settings → display → article icons to be able to hide or show different ones on a small width. The main problem there is probably how to present the settings clearly.

**Reply by Frenzie** [reply]
PS Here's a demo of some potential approach around that https://dev.to/eduferfer/how-to-css-table-swipe-interaction-3ih3

**Reply by ghost** [reply]
TitleWrap is a big improvement, thank you!
Your link for css-table-swipe would be a great option to have in FreshRSS.

**Comment by math-GH** [comment]
Why is it "unusable" in your opinion? I use FreshRSS on my mobile phone regularly.

What needs an improvement?

**Reply by ghost** [reply]
Unusable because the links are too small for me to tap accurately.
It looks fine in the my screen shot above (ironically), but my phone is 5.4‑inch (diagonal). So, difficult to read, difficult to tap and inadequate space between elements. Some responsive CSS would solve this, which I'll look into adding.

**Comment by Alkarex** [comment]
In addition to the custom CSS theme tuning proposed by @Frenzie , FreshRSS is also compatible with several mobile apps proposing various layouts and experiences:
https://github.com/FreshRSS/FreshRSS/blob/edge/README.md#apis--native-apps

**Reply by ghost** [reply]
Thanks. I installed FreshRSS to get away from using dedicated apps entirely, but it's a great feature for those who want it.

**Comment by brihuang99** [comment]
I do think that FreshRSS's mobile web view can benefit greatly from some UI tweaks. Here's a screenshot of Feeder's UI for comparison: 

<img src="https://github.com/FreshRSS/FreshRSS/assets/122472324/bab5ce91-5ab2-407b-a714-a7a0b42924a6" width=50% height=50%>

I do believe that I've seen some issues raised and added in milestones, but  please let me know if I can contribute with designs at all on any open issues. I'm a UI/UX designer and would love to help out!

**Reply by ghost** [reply]
That looks very navigable on my device too. I hope your offer to contribute works out :)

**Reply by Frenzie** [reply]
It might be prudent of me to point out that I dislike many things about that screenshot. I've attached a screenshot of EasyRSS (left) as a counterexample, looking a bit "Android 2" notwithstanding. Though it might also be worth noting that my personal minor FreshRSS style adjustments (right) are nearly 10px taller than the default; I forgot it was.
<img src=https://github.com/FreshRSS/FreshRSS/assets/202757/696b1bc1-ba19-4bcd-a9fe-104508dc3a3c width=45%>  <img src=https://github.com/FreshRSS/FreshRSS/assets/202757/d36d6b82-c6a0-4b67-82e8-9282f243b851 width=45%>

**Reply by ghost** [reply]
It's good that you have text wrapping here, as demonstrated in the top entry "Cure-a Veb Feetels fur Seerch Ingeene-a Oopteemiseshun: Vhet Du Ve-a Need tu Knoo?".
The four columns for icons are the only issue I see with the FreshRSS example.

**Reply by math-GH** [reply]
If you do not like the icon columns then you could hide it with CSS via the CustomCSS extension.
